src/jdk.jartool/share/classes/jdk/security/jarsigner/JarSigner.java

```diff
@@ -76,16 +76,16 @@
  * a {@link NullPointerException}.
  * <p>
  * Example:
- * <pre>
+ * {@snippet lang="java" :
  * JarSigner signer = new JarSigner.Builder(key, certPath)
- *         .digestAlgorithm("SHA-1")
- *         .signatureAlgorithm("SHA1withDSA")
+ *         .digestAlgorithm("SHA-256")
+ *         .signatureAlgorithm("SHA256withRSA")
  *         .build();
  * try (ZipFile in = new ZipFile(inputFile);
  *         FileOutputStream out = new FileOutputStream(outputFile)) {
  *     signer.sign(in, out);
  * }
- * </pre>
+ * }
```

Comment:
This might also be a good opportunity to use the new javadoc @snippet tag (see [JEP 413](url)) which I think has some features to handle the indentation in a consistent manner.

Reply:
Oh, cool, missed that.

Eirik, this issue is currently assigned to @haimaychao, but we can still accept your contribution. Have you or your company signed the OCA?

Reply:
Indented the code as suggested to make it easier to separate from the snippet syntax.

@seanjmullan This is already using the snippet tag :-)

Reply:
@seanjmullan I'm already an OCA sigantory, yes. As a contributor, I have no access to JBS so I'll need with that part. Can you sponsor?